### PR TITLE
Show family name first for Vietnamese

### PR DESCRIPTION
**Pull request by YuanZhu-apple**

### CODE REVIEW

File: ResearchKit/Common/ORKHelpers.m [hunk 1]
```diff
@@ -327,7 +327,7 @@ BOOL ORKCurrentLocalePresentsFamilyNameFirst() {
     static dispatch_once_t onceToken;
```
Comment (rsanchezsaez) [inline ResearchKit/Common/ORKHelpers.m]: Should we remove the space between the `*` and language?

File: ResearchKit/Common/ORKHelpers.m [hunk 2]
```diff
@@ -327,7 +327,7 @@ BOOL ORKCurrentLocalePresentsFamilyNameFirst() {
...
     static NSArray *familyNameFirstLangs = nil;
```
Comment (rsanchezsaez) [inline ResearchKit/Common/ORKHelpers.m]: Should we rename the variable to `familyNameFirstLanguages`? `Lang` is not accepted abbreviation [here](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/APIAbbreviations.html).

**Comment by rsanchezsaez** [comment]
Reviewed. Two nits, otherwise it looks good!

**Comment by rsanchezsaez** [comment]
By the way, I'm [reading](https://en.wikipedia.org/wiki/Family_name#Overview) that the family name also comes first in *Hungarian* and *Romanian*. I don't have access to any natives to check it out. It may be worth confirming to also add those two if that's the case.

**Comment by YuanZhu-apple** [comment]
Maybe not this time for Hungarian and Romanian. Rather wait until there is radar from our QA.

**Comment by oliverschaefer** [comment]
Hi @YuanZhu-apple @rsanchezsaez my mother is Hungarian so I'm at least a 50% native :-). The family name comes first. Hope that helps

**Pull request merged**